action.php

```diff
@@ -28,6 +28,25 @@ public function register(Doku_Event_Handler $controller) {
        $controller->register_hook('FORM_SEARCH_OUTPUT', 'BEFORE', $this, 'addFilterToSearchForm');
        $controller->register_hook('SEARCH_QUERY_FULLPAGE', 'AFTER', $this, 'filterSearchResults');
        $controller->register_hook('SEARCH_QUERY_PAGELOOKUP', 'AFTER', $this, 'filterSearchResults');
+
+        $controller->register_hook('TOOLBAR_DEFINE', 'AFTER', $this, 'handle_toolbar_define');
+    }
+
+
+    /**
+     * Register a new toolbar button
+     *
+     * @param Doku_Event $event event object by reference
+     * @param mixed $param [the parameters passed as fifth argument to register_hook() when this
+     *                           handler was registered]
+     * @return void
+     */
+    public function handle_toolbar_define(Doku_Event $event, $param) {
+        $event->data[] = array(
+            'type' => 'plugin_watchcycle',
+            'title' => 'Add new maintenance syntax',
+            'icon' => DOKU_BASE . 'lib/plugins/watchcycle/images/eye-plus24Black.png',
```

Comment:
I think this should be 16x16 and maybe use the green color to make it look a little bit nicer ;-)